content/doc/book/installing/_kubernetes.adoc

```diff
@@ -552,3 +553,183 @@ This may also be found at:
+The Jenkins Operator is easy to install with just a few manifest and allows 
+users to configure and manage Jenkins on Kubernetes.
+
+===== Prerequisites
+
+Jenkins Operator::
+If you don't have Jenkins Operator installed and configured locally, 
+see the sections below to <<Install Jenkins Operator>>.
+
+===== Install Jenkins Operator
+
+Requirements::
+
+To run Jenkins Operator, you will need:
+
+. Access to a Kubernetes cluster. If you don't have a running Kubernetes cluster, 
+see the link:/doc/book/installing/kubernetes/#create-a-kubernetes-cluster-with-minikube[Create a Kubernetes cluster with minikube] section above.
+
+. kubectl version 1.11+
+
+===== Configure Custom Resource Definition
+
+Custom resources definition (CRD) was introduced in Kubernetes 1.7 and it 
+enables users to add their own/custom objects to the Kubernetes cluster and 
+use it like any other native Kubernetes objects.
+Defining a CRD object creates a new custom resource with a name and schema that you specify. The Kubernetes API serves and handles the storage of your custom resource.
+
+Install Jenkins Custom Resource Definition::
+
+[source,bash]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/crds/jenkins
+----
+
+===== Deploy Jenkins Operator
+There are two ways to deploy the Jenkins Operator.
+
+Using YAML’s::
+
+Apply Service Account and RBAC roles:
+
+[source,bash]
+----
+$ kubectl apply -f https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/deploy/all-in-one-v1alpha2.yaml
+----
+Watch Jenkins Operator instance being created:
+
+[source,bash]
+----
+$ kubectl get pods
+----
+Now Jenkins Operator should be up and running in the default namespace.
+
+Using Helm Chart::
+
+There is an option to use Helm to install the operator. 
+It requires the Helm 3+ for deployment. 
+If you don't have Helm command line interface installed and configured locally, 
+see the sections above to <<Install Helm>>
+
+Create a namespace for the operator::
+See the link:/doc/book/installing/kubernetes/#create-a-namespace[Create a namespace] section above.
+
+Configure Helm::
+Once Helm is installed and set up properly, add the Jenkins-Operator repo as follows:
+
+[source,bash]
+----
+$ helm repo add jenkins https://raw.githubusercontent.com/jenkinsci/kubernetes-operator/master/chart 
+----
+
+Install Jenkins Operator::
+
+[source,bash]
+----
+$ helm install <name> jenkins/jenkins-operator -n jenkins
+----
+
+To add custom labels and annotations, you can use `values.yaml` file as explained in the link:/doc/book/installing/kubernetes/#install-jenkins[Install Jenkins] section above or pass them into helm install command, e.g.:
+
+[source,bash]
+----
+$ helm install <name> jenkins/jenkins-operator -n jenkins --set jenkins.labels.LabelKey=LabelValue,jenkins.annotations.AnnotationKey=AnnotationValue
+----
+
+===== Deploy Jenkins
+Once Jenkins Operator is up and running let’s deploy actual Jenkins instance. 
```

Comment:
`Once Jenkins Operator is up and running let’s deploy actual Jenkins instance.` should read `Once Jenkins Operator is up and running let’s deploy the actual Jenkins instance.`